src/components/HeaderLogo/HeaderLogo.tsx

```diff
@@ -0,0 +1,14 @@
+import { Links } from '../../constants';
+import classes from './headerLogo.module.css';
+
+export const HeaderLogo = () => {
+  return (
+    <a href={`/${Links.HOME.toLowerCase()}`}>
```

Comment:
it's better to create variable for the href link and pass it to the a tag


  const homeLink = Links.HOME.toLowerCase();